### ISSUE TITLE
Make `js.Dictionary` a valid `AttrValue` in `Implicits.scala`

### ISSUE THREAD

**Issue by rpiaggio** [japgolly/scalajs-react#241]
This would avoid the type cast in, for example:

```
^.style := js.Dictionary("background-color" -> "black").asInstanceOf[js.Object]
```

Thank you.

**Comment by japgolly** [comment]
Ah cool. Good idea.

**Comment by japgolly** [comment]
Closed via 3ca600eb255258863f1c8c0f2fbfd8a0df0b142a
